docs/guide/dev/features/mentions/README.md

```diff
@@ -14,4 +14,123 @@ For licensing, see LICENSE.md.
 
 <info-box info="">
     This feature was introduced in CKEditor 4.10. It is provided through optional plugins that are not included in the CKEditor presets available from the <a href="https://ckeditor.com/ckeditor-4/download/">Download</a> site and {@link guide/dev/plugins/README need to be added to your custom build} with <a href="https://ckeditor.com/cke4/builder">online builder</a>.
-</info-box>
\ No newline at end of file
+</info-box>
+
+This plugin provides smart completion feature for custom text matches based on user input. Every time when user type selected marker he will get information about available, existing options.
+
+## Usage
+
+The mentions plugin shows dropdown with available options every time when user types matching text preceded by a **marker** sign.
+
+{@img assets/img/mentions_01.png Using mentions to get user hints.}
+
+When you press <kbd>enter</kbd> or <kbd>tab</kbd> key suggested value will be inserted into an editor.
+
+## Configuration
+
+Mentions plugin is configurable by {@linkapi CKEDITOR.config.mentions global configuration property} or directly when instantiating an editor. Refer to {@link guide/dev/configuration/README Setting CKEditor Configuration} guide for more information about CKEditor configuration.
```

Comment:
Let's simplify it and mention only per-instance configuration. Global config is not the cleanest way to do that (`config.js` would be, which is described in linked docs 👍).